### PR TITLE
Added seqkit/rmdup

### LINKED ISSUE #5701: new module: seqkit/rmdup
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by GallVp**
## PR checklist

Closes #5701 

- [x] Added seqkit/rmdup
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [x] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

**Comment by GallVp** [comment]
Thank you @nvnieuwk 

I can't see you comment.

File: modules/nf-core/seqkit/rmdup/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    tag "seqkit"
+    tag "seqkit/rmdup"
+
+    test("sarscov2-genome_fasta") {
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ], // meta map
+                    file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
```
Comment (nvnieuwk) [inline modules/nf-core/seqkit/rmdup/tests/main.nf.test:18]: Can you update the fetching of the test data to the new [guidelines](https://nf-co.re/docs/guidelines/components/modules#input-data)

**Review by nvnieuwk — COMMENTED** [review]
Sorry it seems I forgot to press save on the comment. Here you have it again :)

**Comment by GallVp** [comment]
Thanks @nvnieuwk 

It is done now!

**Review by nvnieuwk — APPROVED** [review]
LGTM

**Pull request merged**